nuxeo-document-permissions/nuxeo-popup-permission.html

```diff
@@ -284,32 +282,17 @@ <h2>[[_computedTitle]]</h2>
               composed: true,
               bubbles: true,
             }));
-            this.set('params.username', null);
-            this.set('params.email', null);
-            this.set('params.permission', 'Read');
-            this.set('params.begin', null);
-            this.set('params.end', null);
-            this.set('params.notify', true);
-            this.set('params.comment', '');
-            this.set('selectedTimeFrame', this.shareWithExternal ? 'datebased' : 'permanent');
-
+            this.params = this._getResetParams();
+            this.selectedTimeFrame = this.shareWithExternal ? 'datebased' : 'permanent';
           }
         }
 
         computeParams(ace, shareWithExternal) {
-          const params = {
-            username: null,
-            email: null,
-            permission: 'Read',
-            begin: '',
-            end: '',
-            notify: true,
-            comment: '',
-          };
+          const params = this._getResetParams();
 
           // force datebased and notification for external share
           if (shareWithExternal) {
-            this.set('selectedTimeFrame', 'datebased');
+            this.selectedTimeFrame = 'datebased';
```

Comment:
Aren't you sure the `set` wasn't used to notify top-level elements?

Reply:
Yes, it didn't have any notitfy